src/rosdep2/installers.py

```diff
@@ -338,15 +342,15 @@ def resolve(self, rosdep_args):
         """
         packages = None
         if type(rosdep_args) == dict:
```

Comment:
The others are switched to use `isInstance` why not the `dict` here too?

Reply:
Because it is not being reported by `flake8`.